src/main/java/DeadLinks.java

```diff
@@ -1,5 +1,8 @@
+import java.io.IOException;
 import java.util.List;
 
 public interface DeadLinks {
```

Comment:
@dwaynewade0033 what is role of this interface? I thought that he should only return a list of bad links